types/node/tsconfig.json

```diff
@@ -7,7 +7,7 @@
         "module": "commonjs",
         "target": "esnext",
         "lib": [
-            "es6"
+            "es6", "dom"
```

Comment:
is this expected? (adding `dom` back as lib). 
and if yes, should probably be amended in `types/node/ts4.8/tsconfig.json` as well?

Reply:
I posted this PR without working out how to describe it, but the idea is: ts4.8 does not include `"dom"`, so its tests are intended to cover projects that don't include it; the root tsconfig does include `"dom"`, and its tests are intended to cover projects that do include it.

Reply:
Thanks!